### PR TITLE
refactor(docs): yarn upgrade deps

### PR DESCRIPTION
**Pull request by jaketrent**
- Saw some secvuln alerts. Trying to clear those up.

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -76,9 +76,22 @@
     "@pluralsight/ps-design-system-util": "^5.2.0",
     "@pluralsight/ps-design-system-verticaltabs": "^4.1.0",
     "@pluralsight/ps-design-system-viewtoggle": "^6.0.17",
+    "@storybook/addon-actions": "^6.0.21",
```
Comment (jaketrent) [inline docs/package.json:79]: If I did a `cd docs/; yarn upgrade`, why would I have these lines *added*?  I'd think there'd just be modified lines.

Comment (danethurber) [reply docs/package.json:79]: i think they were in the root package.json. i'm not sure either why it put them here

Comment (jaketrent) [reply docs/package.json:79]: maybe I should run yarn upgrade in root instead/first?

**Comment by jaketrent** [comment]
replaced with #1283.

**Pull request closed without merging**